src/pages/[platform]/build-a-backend/restapi/restapi-v5-to-v6-migration-guide/index.mdx

```diff
@@ -0,0 +1,158 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: 'Learn more about the migration steps to upgrade API-Rest APIs from Amplify JavaScript v5 to v6',
+  platforms: ['angular', 'javascript', 'nextjs', 'react', 'vue', 'react-native']
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+Like many of our categories in v6, API (REST) has moved to functional API's in an effort to conform to emerging web standards and maintain a smaller bundle size and better tree-shakability.
+
```

Comment:
functional API's -> functional

> in an effort to conform to emerging web standards and maintain a smaller bundle size and better tree-shakability

The named params unfortunately does not quite achieves these goals, maybe just not mention them?